llvm/lib/CodeGen/MLRegAllocEvictAdvisor.cpp

```diff
@@ -364,6 +364,11 @@ class MLEvictAdvisor : public RegAllocEvictionAdvisor {
 
   using RegID = unsigned;
   mutable DenseMap<RegID, LIFeatureComponents> CachedFeatures;
+
+  mutable std::unordered_map<unsigned, unsigned> VirtRegEvictionCounts;
+  void incrementEvictionCount(const CandidateRegList &Regs,
```

Comment:
nit (sorry for the back and forth) - since it's a const because it only mutates the `mutable`, how about naming the function `onEviction` or something like that - because the incrementing bit is internal book-keeping.

also I meant the function to only wrap the actual incrementing, i.e. `++VirtRegEvictionCounts[...]` bit